packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@freshbooks/api",
-	"version": "2.1.0",
+	"version": "3.0.0",
```

Comment:
There's a process to bump this at release time. Should be fine to leave as-is until then.